packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/models/pin_code_arguments.dart

```diff
@@ -0,0 +1,13 @@
+{{> licence.dart }}
+
+class PinCodeArguments {
+  const PinCodeArguments({
+    required this.title,
```

Comment:
I think that this title property isn't used anywhere, so maybe it could be removed.

Reply:
the title property is used in several places currently such as the update_pin_page, where based on it the navigation to the correct page is happening in the `_isPinCodeVerified` method